systems/registry/node/pkg/server/node_test.go

```diff
@@ -21,28 +21,32 @@ import (
 )
```

Comment:
This file "systems/registry/node/pkg/server/node_test.go " reports few linting errors by GitHub Actions. If these linting errors are not reported in your IDE, make sure you check them on GitHub.

Reply:
fixed